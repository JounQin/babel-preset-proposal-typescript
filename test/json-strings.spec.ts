import _ from './json-strings'

import { execute } from './helpers'

const proposal = 'logical-assignment-operators'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_).toBe('before\u2028after')
})
