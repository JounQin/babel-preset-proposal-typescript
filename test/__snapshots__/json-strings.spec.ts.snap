// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`json-strings 1`] = `"Invalid or unexpected token"`;
